src/modules/ttlmonitor/geometry/Config.cpp

```diff
@@ -75,6 +75,13 @@ void Config::apply() {
     LOG(INIT, Sev::Info, "Using default value for NumberOfMonitors");
   }
   LOG(INIT, Sev::Info, "NumberOfMonitors {}", Parms.NumberOfMonitors);
+
+  try {
+    Parms.MonitorOffset = root["MonitorOffset"].get<int>();
+  } catch (...) {
+    LOG(INIT, Sev::Info, "Using default value for MonitorOffset");
```

Comment:
Shall we give a hint why we use default? We have a try catch here where something failed so maybe would be nice to know. I would throw something which is non blocking or log an error or warning.

Reply:
Nothing failed. This is just a way to support optional configuration vaules. How would you do this instead?

Reply:
I'm afraid that if we use the try catch, to detect optional settings we can actually hide, miss errors. For example incorrect type defined in the json would be overlooked and we would just switched to default value, however there was a value defined.

I'm not an expert on c++ jsons, but I would use  empty() or count() to identify if there is not value and I would use the try any catch to find out real issues.
```c++
if ( !root["node"].empty() ) {
    LOG("Default Value");
```
Also there is a built in default value support for the json library value() function.
https://json.nlohmann.me/api/basic_json/value/